lib/load.php

```diff
@@ -94,6 +94,9 @@ function gutenberg_is_experiment_enabled( $name ) {
 require __DIR__ . '/compat/wordpress-6.2/theme.php';
 require __DIR__ . '/compat/wordpress-6.2/widgets.php';
 
+// WordPress 6.3 compat.
```

Comment:
I believe this could go now after the Tag Processor recently moved to the 6.2 subfolder.